README.md

```diff
@@ -1,9 +1,26 @@
 itkMGHImageIO
 =============
 
-![Build, test, package status](https://github.com/Slicer/itkMGHImageIO/workflows/Build,%20test,%20package/badge.svg)
+![Build, test, package status](https://github.com/InsightSoftwareConsortium/itkMGHImageIO/workflows/Build,%20test,%20package/badge.svg)
 [![PyPI Version](https://img.shields.io/pypi/v/itk-iomgh.svg)](https://pypi.python.org/pypi/itk-iomgh)
 
-A copy of the Slicer itkMGHImageIO mechanism for ITK.
+ITK remote module for IO of images stored in MGH file format.
+
+See https://surfer.nmr.mgh.harvard.edu/fswiki/FsTutorial/MghFormat
+
+## History
+
+The original implementation was developed in the Slicer code base (folder `Slicer/Libs/MGHImageIO`)
+with contributions from Nicole Aucoin, Steve Pieper, Dennis Jen, Sebastien Barre, Jim Miller, Bill Lorensen,
+Dominique Belhachemi, Jean-Christophe Fillion-Robin, Julien Finet, Matt McCormick, Hans Johnson, Kent Williams
+and Andras Lasso.
```

Comment:
I don't have Georghe's last name, but he was a developer at MGH mentioned in a commit message from Nicole, so I'd suggest just adding his name with no last name (maybe he went by just one name?).  Also I'd add Bruce Fischl as a contributor since he helped or at least supervised the integration and interoperability work from the MGH side.

Reply:
Thanks for the review and suggestion. I updated the README to account for this.